internal/controlplane/grpc_accesslog.go

```diff
@@ -99,6 +101,14 @@ func populateLogEvent(
 	switch field {
 	case log.AccessLogFieldAuthority:
 		return evt.Str(string(field), entry.GetRequest().GetAuthority())
+	case log.AccessLogFieldClusterStatName:
+		clusterName := entry.GetCommonProperties().GetUpstreamCluster()
+		if policy := getPolicyByClusterName(srv, clusterName); policy != nil {
```

Comment:
This is potentially an expensive operation, probably too expensive to have here. Is the stat name already available in the upstream cluster properties though?